### PR TITLE
Add reproducible for bitcoinj v0.15.6-rsk-3

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: bitcoinj/0.15.6-rsk-3/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+$ docker build -t bitcoinj/0.15.6-rsk-3 .
+```
+
+## Verify
+
+```
+$ docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
+dea5d3f9e4cee70cde6fca8a26f9d4e975431c4b192ddade899fb0ab3bf34ecc  bitcoinj-core-0.15.6-rsk-3-sources.jar
+b1759d2bd9ff18b9631aa7eb57409fe67749d5d64739cff8b7cc0160bcc0f61a  bitcoinj-core-0.15.6-rsk-3.jar
+580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```
Comment (nathanieliov) [inline bitcoinj/0.15.6-rsk-3/README.md:18]: I got different results:

```
 docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
5c2e0ed68a57803c4a9d11d3889e4fdfd4775a8ad71f9d3f5db7e0896558e5eb  bitcoinj-core-0.15.6-rsk-3-sources.jar
8a720073381d052b888437d7123a39d45dbd2d008839507afec58fec3dded63e  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

May I be doing something wrong?

Comment (julia-zack) [reply bitcoinj/0.15.6-rsk-3/README.md:18]: same here, i got different results for both jars. Also different from natha's

```
docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
5597c66623cce82397fccf361b681cb6bfd7f7de0cf04cd620c3d1c020565b55  bitcoinj-core-0.15.6-rsk-3-sources.jar
8222d818b4491b99e775326adc01dd8af12fbde4040300fbcc70345dc167060f  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

Comment (nathanieliov) [reply bitcoinj/0.15.6-rsk-3/README.md]:
```
> docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
987a6fcacbdfc919c4c826f267c5803195d1770c96c852d7dc6f522ad8c2c7b9  bitcoinj-core-0.15.6-rsk-3-sources.jar
92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

Now, only the sources jar is the one which differs from your result.

File: bitcoinj/0.15.6-rsk-3/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+$ docker build -t bitcoinj/0.15.6-rsk-3 .
+```
+
+## Verify
+
+```
+$ docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
+3bb761ef2b973a678b4181a414441f6b297d79aea5076c7e3c57faddb1847df1  bitcoinj-core-0.15.6-rsk-3-sources.jar
+92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
+580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```
Comment (julia-zack) [inline bitcoinj/0.15.6-rsk-3/README.md:18]: first one is still different

```
docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
db7f6aa4f3ca0f7aa59b7d07850b6d57bf5c0ad8e37c39a38571fe1ff93b4d9f  bitcoinj-core-0.15.6-rsk-3-sources.jar
92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom

```

**Review by nathanieliov — APPROVED** [review]
```
> docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar | grep -v sources.jar'
92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

**Review by julia-zack — APPROVED** [review]

**Review by wilmerrootstock — APPROVED** [review]
```
wilmerrondon@Wilmers-MacBook-Pro 0.15.6-rsk-3 % docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum *'
4ecd02ec81a249b4d44367e2c1a3827ab363c052e35d71d720062194c4a7307e  bitcoinj-core-0.15.6-rsk-3-javadoc.jar
d23bf780d45a27f81a92469af7ea10860a26467eeeaf7ecb9a960b415bf5b97b  bitcoinj-core-0.15.6-rsk-3-sources.jar
92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

**Pull request merged**